shell/platform/android/io/flutter/app/FlutterPluginRegistry.java

```diff
@@ -43,8 +47,10 @@
     private final List<UserLeaveHintListener> mUserLeaveHintListeners = new ArrayList<>(0);
     private final List<ViewDestroyListener> mViewDestroyListeners = new ArrayList<>(0);
 
-    public FlutterPluginRegistry(FlutterNativeView nativeView, Context context) {
+    // NOTE: Either nativeView needs to be null, XOR flutterEngine needs to be null
```

Comment:
uber-nit: end comments in a period.

Reply:
Remove the leading `NOTE: `.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose